tests/std/tests/P0896R4_ranges_alg_fill/test.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+
+struct instantiator {
+    template <class Out>
+    static constexpr void call() {
+        using ranges::fill, ranges::iterator_t;

```

Comment:
`ranges::iterator_t` is unused in this test:
```suggestion
        using ranges::fill;
```